app/src/app/prompt-wars/PromptWars.tsx

```diff
@@ -25,7 +24,7 @@ export const PromptWars: React.FC<PromptWarsProps> = ({ marketId, className }) =
   const [isResultsModalVisible, displayResultsModal] = useState(false);
 
   const { marketContractValues, fetchMarketContractValues, ftTransferCall, sell, create, actions } =
-    useNearPromptWarsMarketContractContext();
+    useEVMPromptWarsMarketContractContext();
```

Comment:
@Udit-takkar I switched to this hook now. 

Running `yarn build` will return errors still, let's focus on fixing them.